pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -977,4 +977,12 @@ self: super: {
   # https://github.com/bos/configurator/issues/22
   configurator = dontCheck super.configurator;
 
+  # Use global libsass library in order to use hsass in the repl
+  hlibsass = overrideCabal super.hlibsass (drv: {
+    configureFlags = drv.configureFlags or [] ++ ["-fexternalLibsass"];
```

Comment:
It seems like this should be the default choice in the generated expression, so I've taken the liberty of changing the generator tool accordingly in https://github.com/NixOS/cabal2nix/commit/f2db36cd0686d17a127d3bea27e402f3473b0083.